socialhome/streams/app/stores/streamStore.operations.js

```diff
@@ -1,29 +1,46 @@
-const stateOperations = {
+import _forEach from "lodash/forEach"
+
+
+const streamStoreOerations = {
```

Comment:
Typo, should be `streamStoreOperations`.

Reply:
Hmm... What's a typo? :D